src/test/java/org/openrewrite/java/micronaut/BeanPropertyCapitalizationStrategyTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 the original author or authors.
+ * Copyright 2023 the original author or authors.
```

Comment:
It's certainly not necessary to always update these numbers; I'd just done so after noticing the licenseHeader.txt was out of date.

Reply:
Makes sense. I hadn't noticed the .txt file was out of date. I noticed though that you've also changed it in the files that already existed for the 2-to-3 recipe - was that intentional? Most projects I've worked on have had a policy of leaving the year to correspond to when a given file was first added to the repository.